Core/Inc/leds/rgb_led_handler.h

```diff
@@ -45,6 +52,7 @@ typedef struct {
     GPIO_PinState blue;
 } rgb_t;
 
-osStatus_t set_rgb_led(cores_t color, control_rgb_led_e control);
+osStatus_t set_rgb_led(const colors_t* pattern, control_rgb_led_e control,
+                       uint8_t sizeOfPattern);
```

Comment:
```suggestion
osStatus_t set_rgb_led(const colors_t* pattern, uint8_t sizeOfPattern, 
control_rgb_led_e control);
```
é mais padrão e intuitivo nessa ordem